taskana-adapter-camunda-system-connector/src/main/java/pro/taskana/adapter/systemconnector/camunda/api/impl/CamundaTaskRetriever.java

```diff
@@ -38,7 +38,7 @@
 
     public List<ReferencedTask> retrieveNewStartedCamundaTasks(String camundaSystemTaskEventUrl) {
 
-        LOGGER.debug("entry to retrieveNewStartedCamundaTasks. createdAfter = {}");
+        LOGGER.debug("### entry to retrieveNewStartedCamundaTasks ###");
```

Comment:
"###" strings in messages are fine during debugging. In Releases / production code they should be removed...

Reply:
Changed